README.md

```diff
@@ -1,2 +1,12 @@
 # BeSIP
 A repository for Be-Secure Improvement proposals
+
```

Comment:
@jobyko  @asa1997  Please go through how the EIP repos has been model ..https://github.com/ethereum/EIPs

Update the readme likewise for BESIP